src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -153,11 +166,18 @@ class WorkspaceDashboardContent extends Component {
             storageCostEstimate
           ])
         ]),
-        div({ style: { margin: '0.5rem 0', borderBottom: `1px solid ${colors.gray[3]}` } }),
+        !_.isEmpty(authorizationDomain) && h(Fragment, [
+          div({ style: styles.header }, ['Authorization Domain']),
+          div({ style: { marginBottom: '0.5rem' } }, [
+            'Collaborators must be a member of all of these groups to access this workspace.'
+          ]),
+          ..._.map(({ membersGroupName }) => div({ style: styles.authDomain }, [membersGroupName]), authorizationDomain)
```

Comment:
TOL: It's generally slightly better to use an explicit array child with keys when you have a dynamic list, as opposed to splicing. In this case it doesn't actually matter, but if the list can change in some way (e.g. filtering), the keys can help react optimize on re-render.